manifests/a/Argotronic/ArgusMonitor/6.2.03/Argotronic.ArgusMonitor.installer.yaml

```
@@ -8,10 +9,9 @@ InstallerType: nullsoft
 UpgradeBehavior: install
 AppsAndFeaturesEntries:
 - ProductCode: ArgusMonitor
```

```suggestion
- ProductCode: ArgusMonitor
  DisplayVersion: 6.2.3.2667
```

Marketing version on the publisher's website is 6.2.03 but version written to Registry (visible in Apps&Features) is 6.2.3.2667